### ISSUE TITLE
Property.prototype.sampleValue doesn't render external $ref properties

### ISSUE THREAD

**Issue by wking** [swagger-api/swagger-ui#721]
I've got [a spec with](https://gist.github.com/wking/9c069f482f11b566880c#file-external-ref-json-L49):

```
"address": {
  "$ref": "http://json-schema.org/address"
}
```

in `definitions.User.properties`.  With a recent Git checkout (187e0a11, updated link, removed buttons, 2014-11-11), the external `$ref` shows up as a link in the “Model” view:

![model](https://cloud.githubusercontent.com/assets/209920/5033933/2d0b3a72-6b1f-11e4-8885-32169c5bb43c.png)

but not in the example JSON of the “Model Schema”:

![model-schema](https://cloud.githubusercontent.com/assets/209920/5033948/49262b40-6b1f-11e4-9568-0356137c9d74.png)

The responsible part of `swagger-client.js` is `Property.prototype.sampleValue`, which has:

```
if(this['$ref']) {
  var refModel = models[this['$ref']];
  if(refModel && typeof ignoredModels[type] === 'undefined') {
    ignoredModels[type] = this;
    output = refModel.getSampleValue(ignoredModels);
  }
  else
    type = refModel;
}
```

That's only trying to lookup the `$ref` locally (in the `models` object), and the `else` block doesn't set a fallback `output` when we don't find it.  I expect the parser should be filling in `models` with parsed versions of external references before we start rendering, but thought I'd create an issue before looking deeper (since I'm new to the codebase).

**Comment by Blechhirn** [comment]
The referenced issue in swagger-js seems to be fixed, closed and from what I can tell working fine. However the UI still keeps showing nothing in the Sample and a URL in the Model view. Should this actually work by now? Swagger Editor seems to be able to display externally referenced definitions.

**Comment by webron** [comment]
@Blechhirn - I'd suggest searching for opened issues, and if you find none, just open a new ticket.
